src/Collapse/Collapse.component.tsx

```diff
@@ -53,9 +55,10 @@ export const Collapse = ({
     isOpen = false,
     openText = DEFAULT_OPEN_TEXT,
     closeText = DEFAULT_CLOSE_TEXT,
-    openIcon = false,
-    closeIcon = false,
+    openIcon,
```

Comment:
The default close and open icons are backwards I think. This isn't related to your PR, just something pre-existing I noticed.

Reply:
Yeah I went back and forth on this. But seeing as how I often make this mistake it probably makes sense to change it.